frontend/src/Components/LibrarySearchResultsModal.tsx

```diff
@@ -235,6 +301,41 @@ const LibrarySearchResultsModal = forwardRef<
                         />
                         <CloseX close={onModalClose} />
                     </div>
+                    {suggestions.length > 0 && (
+                        <div className="relative mt-2 p-2 bg-grey-1 rounded">
+                            <button
+                                onClick={() => setSuggestions([])}
+                                className="btn btn-ghost btn-circle absolute top-1 right-1"
+                            >
+                                x
+                            </button>
+                            <p className="mb-1 text-sm font-semibold">
+                                Did you mean:
+                            </p>
+                            <ul className="flex flex-wrap gap-2">
+                                {suggestions.map((sug, index) => (
+                                    <li key={index}>
+                                        <button
+                                            type="button"
+                                            className="btn btn-sm"
+                                            onClick={() => {
+                                                setSearchTerm(sug);
+                                                setSuggestions([]);
+                                                void handleSearch(1, 10, sug);
+                                            }}
+                                        >
+                                            {sug}
+                                        </button>
+                                    </li>
+                                ))}
+                            </ul>
+                            {suggestionError && (
+                                <p className="text-error text-sm mt-1">
+                                    {suggestionError}
+                                </p>
```

Comment:
I'm on the fence as to whether we display a suggestion error not. It seems like it's a service we're using to enhance the user's experience but do we want to draw attention to it if it doesn't work for some reason?